content/blog/2022-06-14-DVC-VS-Code-extension.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Introducing the DVC extension for Visual Studio Code
+date: 2022-06-14
+description: >
+  Streamline your data science workflow with the DVC extension for VS Code. Data
+  management and experiments all from the comfort of your IDE!
+descriptionLong: >
+  VS Code has become the IDE of choice for many data scientists. Today we are
+  releasing the DVC extension, which will streamline your data science workflow
+  even further. Manage your data, run experiments, compare metrics, and
+  visualize plots, all from the comfort of your IDE.
+# picture: 2022-06-14/header.png
+author: rob_dewit
+# commentsUrl:
+tags:
+  - VS Code
+  - DVC
+  - MLOps
+  - Release
+---
+
+Since its beta release in 2017 DVC has grown to be an essential tool for many
+data science teams. Its data versioning capabilities, reproducible pipelines,
+and experiment tracking features are at the core of our ecosystem of open MLOps
+tools.
+
+Today we are proud to launch a new product that extends the ways in which ML
+teams can use DVC: our extension for Visual Studio Code.
+
+With the VS Code extension, you can control your datasets, run experiments, view
+metrics and plots, and much more. All from your IDE.
+
+[Install the DVC extension from the marketplace to get started.](https://marketplace.visualstudio.com/items?itemName=Iterative.dvc)
+
+# Why an extension?
+
+DVC was built to expand upon the Git workflow. This approach has made it a
+powerful tool that can easily be picked up by users familiar with Git. However,
+for those less familiar with CLI tools, it can be more difficult to get
+accustomed to using DVC. Moreover, while DVC provides many useful
+visualizations, a CLI tool can only take things so far.
+
+With this extension we want to achieve the following:
+
+- Make DVC more accessible by hiding the complexity of the CLI
+- Enhance existing visualizations and providing new ones
+- Move the entire data science workflow into your IDE
+
+# Why Visual Studio Code
+
+One of [our core beliefs at Iterative](https://iterative.ai/why-iterative/) is
+that our products should couply tightly with existing tools and processes for
+data science projects. VS Code is the IDE of choice for many of us at Iterative,
```

Comment:
Would it make sense to also reference stats from some surveys? (i.e. https://insights.stackoverflow.com/survey/2021#integrated-development-environment)

<img width="1311" alt="Captura de Pantalla 2022-05-25 a las 10 57 30" src="https://user-images.githubusercontent.com/12677733/170223994-297969c1-1069-4c6f-b427-4664add020ab.png">